angular/src/app/components/feature-icon/feature-icon.component.html

```diff
@@ -1,6 +1,7 @@
 <div [ngSwitch]="feature.featureType()" style="width: 25px">
     <i class="fas fa-camera-retro" *ngSwitchCase="'image'"></i>
     <i class="fas fa-video" *ngSwitchCase="'video'"></i>
+    <i class="fas fa-question" *ngSwitchCase="'questionnaire'"></i>
```

Comment:
the `survey` type on line 9 seems to not be used anywhere.  i suspect this was Joe and maybe him being aware of the Rapid Questionaire.  Could we drop the `survey case` and use the fa-clipboard-list  instead of the fa-question